autotest/pymod/gdaltest.py

```diff
@@ -1582,6 +1583,28 @@ def skip_on_travis():
     return False
 
 
+###############################################################################
+# Decorator to skip a test if specified creation option is not available
+
+
+def require_creation_option(driver, option):
+
+    md = gdal.GetDriverByName("GTiff").GetMetadata()
```

Comment:
Hard-coded driver name:
```suggestion
    md = gdal.GetDriverByName(driver).GetMetadata()
```
One issue I've noticed is that it seems that @pytest.mark.skipif() method decoration is evaluated before top pytestmark = pytest.mark.require_driver("driver_name"). Consequently, the gdal.GetDriverByName() above might be None. So it should be tested to skip with the reason that the driver is missing if needed

Reply:
Thanks - fixed in 4f2b2c9c8455be08fc90de97b398fae523fe2818